ledfx/config.py

```
@@ -765,6 +765,46 @@ def sanitise_effect_config(effect_type, old_config):
         new_config["audio"]["min_volume"] = 0.2
 
+    # Handle equalizer2d flip_vertical inversion for versions prior to 2.3.6
+    # In 2.3.6, we fixed the upside-down rendering when ring=False and center=False
+    # So we need to invert flip_vertical to maintain the same visual appearance
+    old_version = old_config.get("configuration_version", "0.0.0")
+    if parse_version(old_version) < parse_version("2.3.6"):
+        _LOGGER.warning("Migrating equalizer2d flip_vertical settings for config version < 2.3.6")
+        
+        def invert_equalizer2d_flip_vertical(effect_config):
+            """Helper function to invert flip_vertical for equalizer2d effects"""
+            if (effect_config.get("ring", True) == False and 
```

Ruff (0.13.1)

Avoid equality comparisons to `False`; use `not effect_config.get("ring", True):` for false checks

Replace with `not effect_config.get("ring", True)`

(E712)